pages/common/ionic.md

```diff
@@ -0,0 +1,27 @@
+# ionic
+
+> Ionic project manager and builder.
```

Comment:
We should clarify what this is - "A framework to build hybrid mobile apps"

Reply:
I meant to replace the line with this sentence. Sorry if it wasn't clear. Generally, we avoid using the command name again in the heading. So just "project manager and builder" is not unambiguous enough. Hence my suggestions.

Reply:
Oh sorry, I see what you meant now.